### PR TITLE
Add OV Proposal Op 

### PR DESCRIPTION
**Pull request by LiyangLingIntel**
Add openvino bridge Proposal Op.
Depends on PR [Update Proposal Op #154](https://github.com/plaidml/openvino/pull/154).

### CODE REVIEW

**Review by tzerrell — COMMENTED** [review]
Looks good, minor inline comments

File: plaidml/bridge/openvino/ops/proposal.cpp [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+  }
+  if (attrs.normalize) {
+    x0 = x0 / img_W;
+    y0 = y0 / img_H;
+    x1 = x1 / img_W;
+    y1 = y1 / img_H;
+  }
+
+  // Only keep first -1 in batch indices, set the remaining -1 to be 0
+  // -1 is the flag of termination referring to OV implementation`1
```
[the remark this answers is not in the document]
Comment (LiyangLingIntel) [reply plaidml/bridge/openvino/ops/proposal.cpp]: Yes, it's a typo, fixed.

File: plaidml/bridge/openvino/ops/proposal.cpp [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+  ratios = edsl::reshape(op::tile(ratios, {1, num_scales}), {num_anchors, 1});
+
+  edsl::Tensor base_width, base_height;
+  if (round_ratios) {
+    base_width = edsl::round(edsl::sqrt(edsl::Tensor(base_area) / ratios));
+    base_height = edsl::round(base_width * ratios);
+  } else {
+    base_width = edsl::sqrt(edsl::Tensor(base_area) / ratios);
+    base_height = base_width * ratios;
+  }
```
[the remark this answers is not in the document]
Comment (LiyangLingIntel) [reply plaidml/bridge/openvino/ops/proposal.cpp]: Updated.

**Review by tzerrell — APPROVED** [review]

**Comment by LiyangLingIntel** [comment]
Resolved conflict in `nms` from merging `DetectionOutput`.
@tzerrell Please help review this again, thx.

**Review by tzerrell — APPROVED** [review]

**Pull request merged**